container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -319,9 +319,16 @@ function run_git_tempest {
     pushd $TEMPEST_DIR
 
     discover-tempest-config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
+    && tempest cleanup --init-saved-state || true \
```

Comment:
I think this should be moved before `upload_extra_images` function call and call of discover-tempest-config as it creates some resources sometimes as well (e.g., upload of cirros image).